core-bundle/contao/templates/twig/content_element/accordion_wrapper.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
```

Comment:
I think we could extend the `accordion.html.twig` template here, create a block `accordion_content` there and only overwrite that part. WDYT?

Reply:
I tried that but the `parent()` call inside the `content` block does not allow adding another block.

Reply:
That is true, however, it does not seem to work. 🤔 

<img width="1067" alt="" src="https://github.com/contao/contao/assets/1192057/35452877-32fd-4e51-b60e-20f746cd2883">

Maybe we shouldn‘t extend the `text.html.twig` template?

Reply:
I‘m going to restructure the templates and maybe add a component.

Reply:
Fixed and tested in d73e5794f194bfe46d0e2d11706f9e0d9567ec27.